vm.go

```diff
@@ -474,6 +570,8 @@ func (vm *vm) try(ctx1 context.Context, f func()) (ex *Exception) {
 				ex = &Exception{
 					val: x1,
 				}
+				// TODO: this was a functionality added to match otto but it's causing
+				// an error since x1.ExportType() can be nil
 				if x1.ExportType().Kind() == reflect.String {
```

Comment:
```go
if x1.ExportType().Kind() == reflect.String {
	ex.ignoreStack = true
}
```

but `x1` can be null in certain tests and the returned value from `ExportType()` would be `nil` which would then cause a `panic`. I'm not familiar with much of `goja` internals so I wanted to understand what changes would be ok here.

Reply:
Perfect! I had that change but wasn't sure if it was sufficient, thanks!